generators/client/templates/src/main/webapp/app/services/profiles/_active-profiles.service.js

```diff
@@ -0,0 +1,57 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('<%=angularAppName%>')
+        .factory('ActiveProfiles', ActiveProfiles);
+
+    ActiveProfiles.$inject = ['$q', '$resource'];
+
+    function ActiveProfiles($q, $resource) {
+
+        var _activeProfiles;
+
+        var service = {
+            activeProfiles : activeProfiles
```

Comment:
ok for ProfileService. 

Instead of exposing activeProfiles, should it return 1 response object containing

- activeProfiles
- ribbonEnv (will rename this to ribbonProfile I think)
- inProduction

or do you want individual methods for that on the ProfileService ?